package.json

```diff
@@ -156,8 +156,7 @@
   "eslintIgnore": [
     "node_modules/",
     "lib/",
-    "integration-testing/",
-    "SampleApp/"
+    "integration-testing/"
```

Comment:
Do we want example app to be ignored by es lint similar to sample app?

Reply:
Nope!  I just ignored it previously as I didn't want to go through and fix all the issues in something that I knew was going to be deleted.